gator/hub/__main__.py

```diff
@@ -26,12 +27,16 @@
 @click.option(
     "--static",
     type=click.Path(exists=True, file_okay=False, path_type=Path),
-    default=Path(__file__).parent.parent.parent / "gator-hub",
+    default=Path(
+        os.environ.get(
+            "GATOR_HUB_ROOT", Path(__file__).parent.parent.parent / "gator-hub"
+        )
+    ),
```

Comment:
I think this was the original intention for the env variable (set in pyproject but never used)